server/api/src/rsModAVUMetadata.cpp

```diff
@@ -153,6 +156,8 @@ _rsModAVUMetadata( rsComm_t *rsComm, modAVUMetadataInp_t *modAVUMetadataInp ) {
                                    &modAVUMetadataInp->condInput);
     }
     else if ( strcmp( modAVUMetadataInp->arg0, "adda" ) == 0 ) {
+        logger::api::warn(R"_({} - Use of "adda" is deprecated. Use "add" with ADMIN_KW keyword.)_", __func__);
```

Comment:
Perhaps this message should be added to the rError object. That would allow all clients to see this message.

Reply:
For reference purposes:
https://github.com/irods/irods/blob/3e4ef34f27d3cd62a8885ea1818ac5987e5c5f75/server/api/src/rsDataObjUnlink.cpp#L351-L359

Reply:
Updated